staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -135,6 +135,16 @@ type KubeSchedulerProfile struct {
 	// is scheduled with this profile.
 	SchedulerName *string `json:"schedulerName,omitempty"`
 
+	// PercentageOfNodesToScore is the percentage of all nodes that once found feasible
+	// for running a pod, the scheduler stops its search for more feasible nodes in
+	// the cluster. This helps improve scheduler's performance. Scheduler always tries to find
+	// at least "minFeasibleNodesToFind" feasible nodes no matter what the value of this flag is.
+	// Example: if the cluster size is 500 nodes and the value of this flag is 30,
+	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
+	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
```

Comment:
You should also say that if it's left empty, we will use the top level configuration.

Reply:
Changed it to `It will override global PercentageOfNodesToScore. If it is empty, global PercentageOfNodesToScore will be used.`